commons-extension-webapp/src/main/webapp/ckeditor/config.js

```diff
@@ -1,11 +1,38 @@
 /**
- * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
- * For licensing, see LICENSE.md or http://ckeditor.com/license
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see https://ckeditor.com/legal/ckeditor-oss-license
  */
 
 CKEDITOR.editorConfig = function( config ) {
-	// Define changes to default configuration here. For example:
-	// config.language = 'fr';
-	// config.uiColor = '#AADC6E';
-	config.customConfig = "../eXoConfig.js";
+	// Define changes to default configuration here.
+	// For complete reference see:
+	// https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_config.html
+
```

Comment:
wont this create undesired tool ar buttons?

Reply:
All of eXo modules initialize the editor using this custom config as mentioned [above](https://github.com/Meeds-io/commons/pull/84#discussion_r565122485).

Reply:
ok